modin/experimental/engines/omnisci_on_ray/frame/omnisci_worker.py

```diff
@@ -164,8 +164,7 @@ def put_arrow_to_omnisci(cls, table, name=None):
                 # Conversion for dictionary of null type to string is not supported
                 # in Arrow. Build new column for this case for now.
                 if pa.types.is_null(field.type.value_type):
-                    mask_vals = np.full(table.num_rows, True, dtype=bool)
-                    mask = pa.array(mask_vals)
+                    mask = np.full(table.num_rows, True, dtype=bool)
```

Comment:
since 3.0 it refuses to take pyarrow array as a mask for some reason...
```python
import pyarrow as pa
import numpy as np

print(pa.__version__)

values = [0, 1, 0]
mask_vals = np.array([True, False, True], dtype=bool)
mask = pa.array(mask_vals)

res = pa.array(values, pa.int64(), mask) # pyarrow.lib.ArrowInvalid: Null mask must be NumPy array
print(res)
```